indexers/db/docker-entrypoint-initdb.d/03-schema-consensus.sql

```diff
@@ -282,19 +282,12 @@ CREATE INDEX "consensus_blocks_hash" ON consensus.blocks USING btree (hash);
 CREATE INDEX "consensus_blocks_id_hash" ON consensus.blocks (id, hash);
 CREATE INDEX "consensus_cumulative_blocks_id" ON consensus.cumulative_blocks USING btree (id);
 CREATE INDEX "0xd8db4c8313621519" ON consensus.extrinsics USING btree (id);
-CREATE INDEX "consensus_extrinsics_sort_id" ON consensus.extrinsics USING btree (sort_id DESC);
```

Comment:
Should we replace this with a composite index (block_height, index_in_block)?

Reply:
What do you mean by "normal" indexes?

Reply:
I think a composite index will replace both `sort_id` and `block_height` index and be more performant (both indexing and querying).

Reply:
> Which indexes do you mean by "normal" indexes?

Meaning `single` indexes like these

CREATE INDEX "consensus_extrinsics_sort_id" ON consensus.extrinsics USING btree (sort_id DESC);

Reply:
Yes, it is suggested to have both and see the performance. If it looks stable, slowly remove it, which is not required.

Adding these composite indexes stops the `writes` on the database when indexes are created.

That's why those out of sync were coming. 

added it now